FlowCrypt/Controllers/Setup/SetupCreatePassphraseViewController.swift

```diff
@@ -188,6 +195,48 @@ extension SetupGenerateKeyViewController {
         }
     }
 
+    private func setupAccountWithExistingKeys(with passPhrase: String) {
+        Promise { [weak self] in
+            guard let self = self else { return }
+            self.showSpinner()
+
+            try awaitPromise(self.validateAndConfirmNewPassPhraseOrReject(passPhrase: passPhrase))
+
+            try self.keys.forEach { key in
+                try key.keyDetails.forEach { keyDetail in
+                    guard let privateKey = keyDetail.private else {
+                        throw CreatePassphraseWithExistingKeyError.noPrivateKey
+                    }
+                    let encryptedPrv = try self.core.encryptKey(
+                        armoredPrv: privateKey,
+                        passphrase: passPhrase
+                    )
+                    let parsedKey = try self.core.parseKeys(armoredOrBinary: encryptedPrv.encryptedKey.data())
+                    parsedKey.keyDetails.forEach { keyDetails in
+                        let passPhrase = PassPhrase(value: passPhrase, fingerprints: keyDetails.fingerprints)
+                        self.passPhraseService.savePassPhrase(with: passPhrase, inStorage: self.shouldSaveLocally)
```

Comment:
The pass phrase should only be saved in memory - following the OrgRules. We have already checked that `FORBID_STORING_PASS_PHRASE` is `true` earlier. Now we need to follow it.

Reply:
Should we show a hint to user saying that passphrase will be saved in memory?

Reply:
Good question! Not this time, because we have only one specific customer who has this type of setup, and they will train their stuff to know this. Additionally, in the future, we may allow users to choose temporary vs permanent storage during setup with the EKM flow, similarly how users can choose it in other flows. So I figure it will be easier to add this optionallity if we don't do other adjustments.

Normally, however, this is a good suggestion.